frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.tsx

```diff
@@ -40,6 +41,10 @@ export interface BaseBucketPickerPopoverProps {
   renderTriggerContent: (bucket?: Lib.BucketDisplayInfo) => ReactNode;
   onSelect: (column: Lib.Bucket | NoBucket) => void;
   className?: string;
+  classes?: {
```

Comment:
We usually call these `classNames`, there are no instances where we use the word `classes`

```suggestion
  classNames?: {
```